src/Storages/Distributed/DistributedBlockOutputStream.h

```diff
@@ -62,10 +62,10 @@ class DistributedBlockOutputStream : public IBlockOutputStream
 
     void writeSplitAsync(const Block & block);
 
-    void writeAsyncImpl(const Block & block, const size_t shard_id = 0);
+    void writeAsyncImpl(const Block & block, size_t shard_id = 0);
 
     /// Increments finished_writings_count after each repeat.
-    void writeToLocal(const Block & block, const size_t repeats);
```

Comment:
You forget to update writeToLocal in the module part (.cpp), but in general, what is purpose of this change? (unrelated changes may create unrelated conflicts)

Reply:
`const` in here is unnecessary.